core/controllers/base.py

```diff
@@ -189,9 +189,9 @@ def __init__(self, request, response):  # pylint: disable=super-init-not-called
             if self.user_id is None else user_settings.role)
         self.actions = role_services.get_all_actions(self.role)
 
-        rights_mgr_user = rights_manager.Actor(self.user_id)
-        self.is_moderator = rights_mgr_user.is_moderator()
-        self.is_admin = rights_mgr_user.is_admin()
+        self.is_moderator = bool(
```

Comment:
Should I implement functions like check_moderator and check_admin in role_services.py?

(Yep, admin role can also be given by using actions. But can_create_collections cannot as it will create behaviour change in frontend)